Lucid/Utils/CoreManagerProperty.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CoreManagerProperty.swift
```

Comment:
If you look at the class `CoreManagerSubject` being deleted in this commit (https://github.com/scribd/Lucid/pull/132/commits/78e3133d1c4809bc046af7d65768a9e071ca7c44) you'll see what this is replacing.